components/ListView/DependencyListView.js

```diff
@@ -4,9 +4,7 @@ import PropTypes from "prop-types";
 import ListView from "./ListView";
 import ListItemComponents from "./ListItemComponents";
 
-class DependencyListView extends React.Component {
-  componentWillMount() {}
-
+class DependencyListView extends React.PureComponent {
```

Comment:
You can also use the arrow functions for that.
I am not suggesting it, just so that it's clear.

Reply:
Ahhh so use `const DependencyListView = () => {...}` instead of a PureComponent?

Reply:
You can, just stick to some of the two patterns for the whole project basically :)